datalad/cmdline/helpers.py

```diff
@@ -433,7 +433,7 @@ def _maybe_get_single_subparser(cmdlineargs, parser, interface_groups,
         if not unparsed_args and getattr(parsed_args, 'version', None):
             parsed_args.version()  # will exit with 0
         if not (completing or unparsed_args):
-            fail_handler(parser, msg="too few arguments", exit_code=2)
+            fail_handler(parser, msg="too few arguments, run with --help to get more information.", exit_code=2)
```

Comment:
```suggestion
            fail_handler(
                parser,
                msg="too few arguments, run with --help or visit https://handbook.datalad.org",
                exit_code=2)
```

What about this? Fits in 80 chars on a single line. It has the pointer to `--help` and a URL. Giving the explicit mention of "help" and "handbook" I think we can get away without "to get more information", while still being fairly clear about why one would want to visit this page.

Reply:
```sh
% datalad
error: too few arguments, run with --help or visit https://handbook.datalad.org
usage: datalad [-l LEVEL] [--pbs-runner {condor}] [-C PATH] [--version]
               [--dbg] [--idbg] [-c KEY=VALUE]
               [-f {default,json,json_pp,tailored,'<template>'}]
               [--report-status {success,failure,ok,notneeded,impossible,error}]
               [--report-type {dataset,file}]
               [--on-failure {ignore,continue,stop}] [--cmd] [-h]
               command [command-opts]
```

Reply:
Ok, let's go with this